host/sgx/create.c

```diff
@@ -357,6 +363,10 @@ static oe_result_t _calculate_enclave_size(
         image_size + heap_size +
         (size_settings->num_tcs * (stack_size + tls_size + control_size));
 
+    /* Check if 0-base enclave and not simulated*/
```

Comment:
Should we just ignore/unset the `create_zero_base_enclave` flag in the simulation mode, and throw a message says it's not supported?

Reply:
I have made it raise an error now saying 0-base is not supported in simulation mode. Do you prefer we choose the usual path, ignoring the request?